src/path_handler.cpp

```diff
@@ -19,7 +19,7 @@ void PathHandler::initialize(
   logger_ = node->get_logger();
 
   auto getParam = utils::getParamGetter(node, name_);
-  getParam(max_robot_pose_search_dist_, "max_robot_pose_search_dist", getMaxCostmapDist());
+  getParam(max_robot_pose_search_dist_, "max_robot_pose_search_dist", 1.2);
```

Comment:
Back to `getMaxCostmapDist()`?

Reply:
If one use GoalCritic or PathAngleCritic, this value could greatly change behaviour, that's why i create new critics inside reference critic, and make this default value a constant

Reply:
It shouldn't, its only used to find the upper-bound of positions to search for the nearest robot pose https://github.com/artofnothingness/mppic/blob/edf87e1b451a721eb68a393943912a774857ce2c/src/path_handler.cpp#L33-L35. It has nothing to do with the actual pruning of the path, just to find where the robot is relative to a path. This should match the costmap size by default so that we can check in the local time horizon. The only time this would be problematic is if there were loops in the path within the local costmap bounds

Reply:
Got it. I'll bring that back. It was lookahead distance that limit upper bound, i'll return that too

Reply:
I'd use `prune_distance` instead of look ahead distance, since this is the `path handler` object and its going to prune the path by that distance. That's more semantically meaningful

Reply:
done